leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/RedisRegistrationStore.java

```diff
@@ -62,6 +62,11 @@
  */
 public class RedisRegistrationStore implements CaliforniumRegistrationStore, Startable, Stoppable {
 
+    /** Default time in seconds between 2 cleaning tasks (used to remove expired registration). */
+    public static final long DEFAULT_CLEAN_PERIOD = 60;
+    /** Defaut Extra time for registration lifetime in seconds */
+    public static final long DEFAULT_CLEMENCY = 0;
```

Comment:
what do you think of renaming "clemency" to "grace period"? again a naming question ;)

Reply:
I will change it